test.py

```diff
@@ -1279,8 +1279,9 @@ def parse_cmd_line() -> argparse.Namespace:
                 suites. Each name is used as a substring to look for in the
                 path to test file, e.g. "mem" will run all tests that have
                 "mem" in their name in all suites, "boost/mem" will only enable
-                tests starting with "mem" in "boost" suite. Default: run all
-                tests in all suites.""",
+                tests starting with "mem" in "boost" suite, and
+                "boost/memtable_test::test_hash_is_cached" to narrow down to
+                a certain test case. Default: run all tests in all suites.""",
```

Comment:
Thanks for doing this. Just today I struggled to find the right syntax to run a **single** test (I refuse to call the whole file "a test", sorry ;-)), and failed. I forgot it is necessary to include the suite prefix (which isn't necessary when running a whole test file) :-(

Reply:
Aha, same here yesterday morning!